Bottom: 7ba34a43b4e207a8200d498d48e39603515f2edf
Top:    bc8b701a9115c48356c09dc995d1252c85c31355
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:11 +0000

Turn on thindraw poly eye-candy


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 3b54c6e..8229e63 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -904,7 +904,7 @@ ghid_thindraw_pcb_polygon (hidGC gc, PolygonType *poly, const BoxType *clip_box)
 
   double old_alpha_mult = gtk_gc->alpha_mult;
   common_thindraw_pcb_polygon (gc, poly, clip_box);
-#if 0
+#if 1
   ghid_set_alpha_mult (gc, gtk_gc->alpha_mult * 0.25);
   hid_draw_fill_pcb_polygon (gc, poly, clip_box);
   ghid_set_alpha_mult (gc, old_alpha_mult);
